willow/plugins/wand.py

```diff
@@ -164,8 +170,18 @@ def save_as_jpeg(
         :param apply_optimizers: controls whether to run any configured optimizer libraries
         :return: JPEGImageFile
         """
-        with self.image.convert("pjpeg" if progressive else "jpeg") as converted:
+        with self.image.clone() as converted:
```

Comment:
@andre-fuchs I changed this back to use `convert` as https://docs.wand-py.org/en/latest/wand/image.html#wand.image.Image.convert as that returns a new image, rather than do anything on the original